packages/service-backend/src/course/course.service.ts

```diff
@@ -37,6 +39,27 @@ export class CourseService {
     ctx: any,
   ): Promise<CourseIdResponse> {
     try {
+      const userSubscription = await prisma.userSubscription.findFirst({
+        where: {
+          user_id: ctx.__user.id,
+        },
+      });
+
+      if (!userSubscription) {
+        const basicSubscription = await prisma.subscription.findFirst({
+          where: {
+            plan: SubscriptionPlan.BASIC,
+          }
+        });
```

Comment:
You should be able to create a subscription by the name and don't do an useless request to the database you already know the plan that you want to add: `SubscriptionPlan.BASIC`

Be sure that when a user is created the SubscriptionPlan.BASIC is added with it.